src/main/java/at/ac/tuwien/student/sese2017/xp/hotelmanagement/domain/test/TestDataV2.java

```diff
@@ -0,0 +1,26 @@
+package at.ac.tuwien.student.sese2017.xp.hotelmanagement.domain.test;
+
+
+/**
+ * Version 2 of the test data.
+ * <p>
+ *   Object wrapper for test data instances
+ * </p>
+ */
+public class TestDataV2 extends TestDataV1 {
```

Comment:
Das naming scheme wird nicht standhalten, wenn wir parallel in verschiedenen branches arbeiten.
Bspw. pair A entwickelt feature 1 und pair B entwickelt feature 2. Beide pairs verwenden den TestDataInjectorV2 als Basis und nennen ihren TestDataInjectorV3.
Das führt dann 1. zu merge conflicts und 2. nur wieder zur ungewollten Modifizierung der Testdaten